index.md

```diff
@@ -25,11 +25,15 @@ Alum who can help?
 ------------------
 
 If you would like to offer support to a student in need, fill out [this
-form](#). You will be able to specify what kind of support and how much
+form](). You will be able to specify what kind of support and how much
```

Comment:
I don't think I meant to make this change, I'm assuming this would be a separate form?

Reply:
Yeah, it's a separate form that I should totally make and push. Can you revert this part of the change?